{"name": "ввод почты и клик по кнопке Восстановить»", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_NAME_NOT_RESOLVED\n  (Session info: chrome=130.0.6723.70)\nStacktrace:\n\tGetHandleVerifier [0x00A838B3+24035]\n\t(No symbol) [0x00A0BC44]\n\t(No symbol) [0x008EC2D3]\n\t(No symbol) [0x008EA0F9]\n\t(No symbol) [0x008DDED8]\n\t(No symbol) [0x008DF80B]\n\t(No symbol) [0x008DE127]\n\t(No symbol) [0x008DDACC]\n\t(No symbol) [0x008DD9F8]\n\t(No symbol) [0x008DBE67]\n\t(No symbol) [0x008DC3EB]\n\t(No symbol) [0x008EECAE]\n\t(No symbol) [0x0096A017]\n\t(No symbol) [0x0094FE8C]\n\t(No symbol) [0x0096953F]\n\t(No symbol) [0x0094FC26]\n\t(No symbol) [0x0092218C]\n\t(No symbol) [0x0092310D]\n\tGetHandleVerifier [0x00D296D3+2800643]\n\tGetHandleVerifier [0x00D8428E+3172286]\n\tGetHandleVerifier [0x00D7CEA2+3142610]\n\tGetHandleVerifier [0x00B26C60+692624]\n\t(No symbol) [0x00A14C5D]\n\t(No symbol) [0x00A11968]\n\t(No symbol) [0x00A11B00]\n\t(No symbol) [0x00A03FB0]\n\tBaseThreadInitThunk [0x756E7BA9+25]\n\tRtlInitializeExceptionChain [0x772DC0CB+107]\n\tRtlClearBits [0x772DC04F+191]", "trace": "self = <test_forgot_password_page.TestForgotPasswordPage object at 0x00000275757700E0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"758ddef58144c39c5891804ac4c503d3\")>\n\n    @allure.title(\"ввод почты и клик по кнопке Восстановить»\")\n    def test_enter_email_and_click_on_the_restore_button(self, driver):\n        forgot_password_page = ForgotPasswordPage(driver)\n>       forgot_password_page.opening_the_login_page()\n\ntests\\test_forgot_password_page.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npage_objects\\forgot_password_page.py:24: in opening_the_login_page\n    self.open_page(self.login_url)\npage_objects\\base_page.py:25: in open_page\n    self.driver.get(url)\n..\\..\\qa_python_1\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:363: in get\n    self.execute(Command.GET, {\"url\": url})\n..\\..\\qa_python_1\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:354: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x000002757483E210>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_NAME_NOT_RESOLVED\\\\n  (...dInitThunk [0x756E7BA9+25]\\\\n\\\\tRtlInitializeExceptionChain [0x772DC0CB+107]\\\\n\\\\tRtlClearBits [0x772DC04F+191]\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_NAME_NOT_RESOLVED\nE         (Session info: chrome=130.0.6723.70)\nE       Stacktrace:\nE       \tGetHandleVerifier [0x00A838B3+24035]\nE       \t(No symbol) [0x00A0BC44]\nE       \t(No symbol) [0x008EC2D3]\nE       \t(No symbol) [0x008EA0F9]\nE       \t(No symbol) [0x008DDED8]\nE       \t(No symbol) [0x008DF80B]\nE       \t(No symbol) [0x008DE127]\nE       \t(No symbol) [0x008DDACC]\nE       \t(No symbol) [0x008DD9F8]\nE       \t(No symbol) [0x008DBE67]\nE       \t(No symbol) [0x008DC3EB]\nE       \t(No symbol) [0x008EECAE]\nE       \t(No symbol) [0x0096A017]\nE       \t(No symbol) [0x0094FE8C]\nE       \t(No symbol) [0x0096953F]\nE       \t(No symbol) [0x0094FC26]\nE       \t(No symbol) [0x0092218C]\nE       \t(No symbol) [0x0092310D]\nE       \tGetHandleVerifier [0x00D296D3+2800643]\nE       \tGetHandleVerifier [0x00D8428E+3172286]\nE       \tGetHandleVerifier [0x00D7CEA2+3142610]\nE       \tGetHandleVerifier [0x00B26C60+692624]\nE       \t(No symbol) [0x00A14C5D]\nE       \t(No symbol) [0x00A11968]\nE       \t(No symbol) [0x00A11B00]\nE       \t(No symbol) [0x00A03FB0]\nE       \tBaseThreadInitThunk [0x756E7BA9+25]\nE       \tRtlInitializeExceptionChain [0x772DC0CB+107]\nE       \tRtlClearBits [0x772DC04F+191]\n\n..\\..\\qa_python_1\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py:229: WebDriverException"}, "steps": [{"name": "Открытие страницы авторизации", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_NAME_NOT_RESOLVED\n  (Session info: chrome=130.0.6723.70)\nStacktrace:\n\tGetHandleVerifier [0x00A838B3+24035]\n\t(No symbol) [0x00A0BC44]\n\t(No symbol) [0x008EC2D3]\n\t(No symbol) [0x008EA0F9]\n\t(No symbol) [0x008DDED8]\n\t(No symbol) [0x008DF80B]\n\t(No symbol) [0x008DE127]\n\t(No symbol) [0x008DDACC]\n\t(No symbol) [0x008DD9F8]\n\t(No symbol) [0x008DBE67]\n\t(No symbol) [0x008DC3EB]\n\t(No symbol) [0x008EECAE]\n\t(No symbol) [0x0096A017]\n\t(No symbol) [0x0094FE8C]\n\t(No symbol) [0x0096953F]\n\t(No symbol) [0x0094FC26]\n\t(No symbol) [0x0092218C]\n\t(No symbol) [0x0092310D]\n\tGetHandleVerifier [0x00D296D3+2800643]\n\tGetHandleVerifier [0x00D8428E+3172286]\n\tGetHandleVerifier [0x00D7CEA2+3142610]\n\tGetHandleVerifier [0x00B26C60+692624]\n\t(No symbol) [0x00A14C5D]\n\t(No symbol) [0x00A11968]\n\t(No symbol) [0x00A11B00]\n\t(No symbol) [0x00A03FB0]\n\tBaseThreadInitThunk [0x756E7BA9+25]\n\tRtlInitializeExceptionChain [0x772DC0CB+107]\n\tRtlClearBits [0x772DC04F+191]\n\n", "trace": "  File \"C:\\Users\\79168\\PycharmProjects\\qa_python_1\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\79168\\PycharmProjects\\Diplom\\Diplom_3\\page_objects\\forgot_password_page.py\", line 24, in opening_the_login_page\n    self.open_page(self.login_url)\n  File \"C:\\Users\\79168\\PycharmProjects\\Diplom\\Diplom_3\\page_objects\\base_page.py\", line 25, in open_page\n    self.driver.get(url)\n  File \"C:\\Users\\79168\\PycharmProjects\\qa_python_1\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 363, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\79168\\PycharmProjects\\qa_python_1\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\79168\\PycharmProjects\\qa_python_1\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1730476867778, "stop": 1730476868001}], "parameters": [{"name": "driver", "value": "'chrome'"}], "start": 1730476867776, "stop": 1730476868003, "uuid": "c753f23b-cc13-4a5b-a7f9-85ad32f3e6a3", "historyId": "7d330abb26935955ff1f6588154c4db6", "testCaseId": "3b489a5cff252cff30602a086522dc01", "fullName": "tests.test_forgot_password_page.TestForgotPasswordPage#test_enter_email_and_click_on_the_restore_button", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_forgot_password_page"}, {"name": "subSuite", "value": "TestForgotPasswordPage"}, {"name": "host", "value": "LAPTOP-G5OQRFSD"}, {"name": "thread", "value": "8500-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_forgot_password_page"}]}